app/assets/javascripts/factories/repository/repository_data.js

```diff
@@ -0,0 +1,18 @@
+ManageIQ.angularApplication.factory('repositoryDataFactory', ['$http', function($http) {
+    var factoryRepository = {
```

Comment:
It's not wrong, just surprised me, I'm used to using services more.. but I don't really object to factories :). And yes, SSUI calls them services but inside they are mostly factories :).